scripts/sklearn_training_trees.py

```diff
@@ -213,7 +213,10 @@
 
 if args.out:
    log.info('Dumping training file in: ' + args.out)
-   joblib.dump(clf, args.out, compress=True) 
+   # *** Sklearn(python)-type training file (.pkl) ***
+   joblib.dump(clf, args.out, compress=True)
+   # *** TMVA-style training file (.xml) ***
+   #convert.gbr_to_tmva(clf,X,TMVAClassification_BDTG.weights.xml,mva_name = "BDTG",coef = 10, var_names = variables)
```

Comment:
instead of commenting/uncommenting you could do:
   1. add a switch that allows saving the weight in xml together with the .pkl (BTW, TMVAClassification_BDTG.weights.xml should go between quotes)
   2. Let the system choose, if args.out is "something.pkl" go for pickle, if it is 'something.xml' go for the conversion